airflow/providers/databricks/operators/databricks.py

```diff
@@ -536,6 +542,12 @@ def _get_hook(self) -> DatabricksHook:
 
     def execute(self, context: 'Context'):
         hook = self._get_hook()
+        if 'job_name' in self.json:
+            job_id = hook.find_job_id_by_name(self.json['job_name'])
+            if not job_id:
```

Comment:
Are we sure that the job id in Databricks can never be `0`? In that case `not job_id` will be evaluated to `True`. So probably a `job_id is None` could be a better approach.